integration-tests/vitest/vitest.spec.js

```diff
@@ -203,5 +204,34 @@ versions.forEach((version) => {
         )
       })
     })
+
+    it('correctly calculates test code owners when working directory is not repository root', (done) => {
+      const eventsPromise = receiver
+        .gatherPayloadsMaxTimeout(({ url }) => url.endsWith('/api/v2/citestcycle'), (payloads) => {
+          const events = payloads.flatMap(({ payload }) => payload.events)
+
+          const test = events.find(event => event.type === 'test').content
+          assert.equal(test.meta[TEST_CODE_OWNERS], JSON.stringify(['@datadog-dd-trace-js']))
```

Comment:
ℹ️ vitest's `test.suite` is the same as `test.source.file`, so we don't check whether they're different. The reason why this is not the case for other frameworks is backwards compatibility, which does not apply for vitest since its support is recent